api/envoy/config/trace/v3/opentelemetry.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package envoy.config.trace.v3;
+
+import "envoy/config/core/v3/grpc_service.proto";
+
+import "udpa/annotations/migrate.proto";
+import "udpa/annotations/status.proto";
+import "udpa/annotations/versioning.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.config.trace.v3";
+option java_outer_classname = "OpentelemetryProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/config/trace/v3;tracev3";
+option (udpa.annotations.file_migrate).move_to_package =
+    "envoy.extensions.tracers.opentelemetry.v4alpha";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: OpenTelemetry tracer]
+
+// Configuration for the OpenTelemetry tracer.
+//  [#extension: envoy.tracers.opentelemetry]
+message OpenTelemetryConfig {
+  // The upstream gRPC cluster that will receive OTLP traces. Only Google gRPC is supported.
```

Comment:
I don't see the reason that only Google gRPC is supported, why is that?

Reply:
I think I copied this from the other comment as well, but after taking another look, I also don't think this applies. Removed this warning.